steps/infra/aws/main.tf

```diff
@@ -41,26 +41,6 @@ module "masters" {
   user_data_igns      = "${var.tectonic_ignition_masters}"
 }
 
-module "workers" {
-  source = "../../../modules/aws/worker"
```

Comment:
You should remove the referenced worker module as well; no need for it anymore :).

Reply:
Done.

Reply:
> You should remove the referenced worker module as...

Done in 3d5422b5a3ba.